MANIFEST.in

```diff
@@ -15,4 +15,3 @@ recursive-include src *.py
 recursive-include docs *.ast
 recursive-include docs *.bat
 recursive-include docs *.jpg
-recursive-include tests *.py
```

Comment:
This file is generated, removing this line has to be changed in `.meta.toml`, too.